server/index.js

```diff
@@ -1,24 +1,127 @@
+/* eslint-disable guard-for-in */
+/* eslint-disable camelcase */
+/* eslint-disable no-console */
+/* eslint-disable no-restricted-syntax */
 const express = require('express');
-const db = require('./database/db.js');
 const path = require('path');
+const moment = require('moment');
+const db = require('./database/db.js');
 
 const app = express();
 
+app.use((req, res, next) => {
+  console.log(`serving ${req.method} for ${req.url}`);
+  next();
+});
+
 app.use(express.json());
 app.use('/hostels/:hostel_id', express.static(path.join(__dirname, '../public')));
 
-
 app.get('/hostels/:id/api/reviews', (req, res) => {
   const queryStr = `SELECT * FROM reviews INNER JOIN authors ON reviews.author_id = authors.id WHERE reviews.hostel_id = ${req.params.id}`;
   db.connection.query(queryStr, (err, response) => {
     if (err) {
-      console.log(err);
+      console.log(`an error occured getting all reviews for hostel id ${req.params.id}`, err);
       res.sendStatus(500);
     } else {
-      console.log('done');
+      console.log('query successful');
       res.json(response);
     }
   });
 });
 
+app.get('/api/reviews/:id', (req, res) => {
+  const queryStr = `SELECT * FROM reviews INNER JOIN authors ON reviews.author_id = authors.id WHERE reviews.id = ${req.params.id}`;
+  db.connection.query(queryStr, (err, response) => {
+    if (err) {
+      console.log(`an error occured getting review id ${req.params.id}`, err);
+      res.sendStatus(500);
+    } else {
+      console.log('query successful');
+      res.json(response);
```

Comment:
Not required, but something to think about: maybe you could move all these DB queries to a models folder/file?

Do you also want to send back a custom error message letting the user know what went wrong?

Reply:
I definitely thought about both of these but didn't want to put the work in since I'm about to delete it.